components/search.py

```diff
@@ -138,10 +171,9 @@ def fetch_docs(self) -> None:
                         )
                     )
 
-    def fetch_wiki(self) -> None:
-        request = Request(WIKI_URL, headers={"User-Agent": USER_AGENT})
-        with urlopen(request) as file:
-            wiki_soup = BeautifulSoup(file, "html.parser")
+    async def update_wiki(self) -> None:
+        response = await self._httpx_client.get(url=WIKI_URL, headers={"User-Agent": USER_AGENT})
```

Comment:
```suggestion
        response = await self._httpx_client.get(url=WIKI_URL, headers=self.HEADERS)
```
maybe extract this as class variable?